File: SV.Web/src/app/shared/types/api/card-detail-response.ts
import { CardPack } from './../customs/card-pack.enum';
import { EvoSpecs } from '../customs/evo-specs';
import { BattleStats } from './../customs/battle-stats';
import { CardResponse } from './card-response';

export interface CardDetailResponse extends CardResponse {
  abilityText: string;
  audioLocations: string[];
  battleStats: BattleStats;
  cardPack: CardPack;
  evo: EvoSpecs;
  flavorText: string;
  ppCost: number;
}

File: SV.Edge/src/SV.Edge/Controllers/Models/CardDetailResponse.cs
using SV.Edge.Services.Models;

namespace SV.Edge.Controllers.Models;

public class CardDetailResponse : CardResponse
{
    public string AbilityText { get; init; }
    public IList<string> AudioLocations { get; init; }
    public BattleStats BattleStats { get; init; }
    public string CardPack { get; init; }
    public EvoSpecs Evo { get; init; }
    public string FlavorText { get; init; }
    public int PPCost { get; init; }
}